### PR TITLE
Added table of possible values for setting shortcuts

### PR DESCRIPTION
**Pull request by bomberstudios**

### CODE REVIEW

File: docs/pages/plugins-manifest.md [hunk 1]
```diff
@@ -106,11 +106,37 @@ function selectAll() {
...
 | `handlers`    | If more fine-grain control is required use the `handlers` field instead of `handler`, see below for in-depth documentation                                                                                                                                                                                                                                                                                                                                                                                        |
 
+##### Shortcut Values
+
+When setting the shortcut for a command, you can use any combination of the following keys, plus any regular character. Keep in mind that some characters may require users to press modifier keys on some international keyboard layouts. For example, to get `/` on a US keyboard you just need to press one key, but on a Spanish keyboard you need to press `shift 7` to get the same character. So a command with a `ctrl shift /` shortcut can't be triggered on a Spanish keyboard, because it gets interpreted as `ctrl /`.
```
Comment (christianklotz) [inline docs/pages/plugins-manifest.md]: I'd suggest to make it more concise and move the edge case explanation to a separate paragraph and start with the recommendation first before backing it up with an example, i.e. _"test with international keyboard layouts, here's why"_:

> Command shortcuts can set using any combination of the following keys, plus any regular character.
> Make sure to test your shortcuts with different international keyboard layouts. Certain characters can require users to press modifier keys. For example a shortcut like `shift /` would **not** work with a Spanish keyboard layout. Whereas a forward slash `/` has a dedicated key on a US keyboard, it requires pressing `shift 7` on a Spanish keyboard.

Comment (bomberstudios) [reply docs/pages/plugins-manifest.md]: Much better, yes. Will change it 🙌

File: docs/pages/plugins-manifest.md [hunk 2]
```diff
@@ -113,7 +113,14 @@ function selectAll() {
 
 ##### Shortcut Values
 
-When setting the shortcut for a command, you can use any combination of the following keys, plus any regular character. Keep in mind that some characters may require users to press modifier keys on some international keyboard layouts. For example, to get `/` on a US keyboard you just need to press one key, but on a Spanish keyboard you need to press `shift 7` to get the same character. So a command with a `ctrl shift /` shortcut can't be triggered on a Spanish keyboard, because it gets interpreted as `ctrl /`.
+Command shortcuts can be set using any combination of the following keys, plus any regular character.
+
+Shortcuts need to use at least one modifier key (Command, Option, Control or Shift), you cannot define single key shortcuts in `manifest.json`.
+
+If your command tries to use any of the predefined Sketch shortcuts (say, `cmd s`), the shortcut will be ignored and pressing those keys will perform the default action in Sketch.
```
Comment (christianklotz) [inline docs/pages/plugins-manifest.md]: Somewhat related to previous comment, I think reducing the "conversational" style can help make it more concise.

> Please note, values matching predefined Sketch shortcuts, e.g. `cmd s` will be ignored.

Comment (christianklotz) [reply docs/pages/plugins-manifest.md]: > Please note, shortcuts must contain at least on modifier key, _Command_, _Option_, _Control_ or _Shift_. Values matching predefined Sketch shortcuts, e.g. cmd s will be ignored.
